frontend/src/interfaces/types.ts

```diff
@@ -1,36 +1,36 @@
-import { NavigateFunction } from 'react-router-dom'
+import { NavigateFunction } from 'react-router-dom';
 
 export interface RegisterData {
-  email: string
-  password: string
-  firstName: string
-  lastName: string
+  email: string;
+  password: string;
+  firstName: string;
+  lastName: string;
 }
 
 export interface LoginData {
-  email: string
-  password: string
+  email: string;
+  password: string;
 }
 export interface ContextTypes {
-  user: UserType | null | undefined | {}
+  user: UserType | null | undefined;
   login: (
     data: LoginData,
     onSuccess: (nav: NavigateFunction) => void,
     onError: (error: string) => void
-  ) => void
+  ) => void;
   register: (
     data: RegisterData,
     onSuccess: () => void,
     onError: (error: string) => void
-  ) => void
-  loading: boolean
-  accessToken: string
+  ) => void;
+  loading: boolean;
+  accessToken: string;
 }
 export interface UserType {
-  id: number
-  firstName: string
-  lastName: string
-  email: string
-  jwtToken: string
-  role: number
+  id: number;
+  firstName: string;
+  lastName: string;
+  email: string;
+  jwtToken: string;
+  role: number;
```

Comment:
Isn't this a Role enum?

Reply:
The backend returns a number, but setting this as a Role enum should work as well, I cannot test it on this branch because the Login doesn't work.